app/controllers/WorkerController.php

```diff
@@ -48,4 +66,31 @@ public function logoutAction()
 
         return $this->response->redirect(['for' => 'home'], false, 200);
     }
+
+    /**
+     * If Admin is logged in, registers new worker by calling validation and if ok,
+     * calls Workers model's register method
+     */
+    public function registerAction()
+    {
+        if (false === $this->workerService->isLoggedIn()) {
+            $this->response->redirect(['for' => 'home'], false, 403);
+        }
+
+        if (null !== $this->request->getPost('submit')) {
+            $worker = $this->request->getPost();
+            $messages = $this->workerValidator->validate($worker);
+
+            if (0 === $messages->count()) {
+                $worker['password'] = $this->workerService->hashPassword($worker['password']);
+
+                if (true === $this->workerModel->register($worker)) {
+                    $this->flashSession->success('Worker registration successful');
+                    $this->response->redirect(['for' => 'registerWorker'], false, 200);
```

Comment:
there are so many redirects in this project... and from what i see 2nd and 3rd param are mostly all the time false and 200.
think to make this common usage as default ussage